fiftyone/plugins/definitions.py

```diff
@@ -86,7 +86,9 @@ def fiftyone_requirement(self):
     @property
     def operators(self):
         """The operators of the plugin."""
-        return self._metadata.get("operators", [])
+        return self._metadata.get("operators", []) + self._metadata.get(
```

Comment:
@ritch just confirming: this adds support for `panels` as an alias of `operators` in `fiftyone.yml` files